metadata-ingestion/tests/integration/great-expectations/ge_mcps_golden.json

```diff
@@ -1,782 +1,1929 @@
+        }
+    }
 },
 {
-    "auditHeader": null,
     "entityType": "assertion",
     "entityUrn": "urn:li:assertion:6a54a953afa6403d7b6dffbf02ff3de4",
-    "entityKeyAspect": null,
     "changeType": "UPSERT",
     "aspectName": "dataPlatformInstance",
     "aspect": {
-        "value": "{\"platform\": \"urn:li:dataPlatform:great-expectations\"}",
-        "contentType": "application/json"
-    },
-    "systemMetadata": null
+        "json": {
+            "platform": "urn:li:dataPlatform:great-expectations"
+        }
+    }
 },
 {
-    "auditHeader": null,
     "entityType": "assertion",
     "entityUrn": "urn:li:assertion:6a54a953afa6403d7b6dffbf02ff3de4",
-    "entityKeyAspect": null,
     "changeType": "UPSERT",
     "aspectName": "assertionRunEvent",
     "aspect": {
-        "value": "{\"timestampMillis\": 1640692800000, \"partitionSpec\": {\"type\": \"FULL_TABLE\", \"partition\": \"FULL_TABLE_SNAPSHOT\"}, \"runId\": \"2021-12-28T12:00:00Z\", \"assertionUrn\": \"urn:li:assertion:6a54a953afa6403d7b6dffbf02ff3de4\", \"asserteeUrn\": \"urn:li:dataset:(urn:li:dataPlatform:postgres,postgres1.postgres.public.foo2,PROD)\", \"batchSpec\": {\"customProperties\": {\"data_asset_name\": \"foo2\", \"datasource_name\": \"my_postgresql_datasource\"}, \"nativeBatchId\": \"010ef8c1cd417910b971f4468f024ec5\"}, \"status\": \"COMPLETE\", \"result\": {\"type\": \"SUCCESS\", \"rowCount\": 5, \"unexpectedCount\": 0, \"nativeResults\": {}}}",
-        "contentType": "application/json"
-    },
-    "systemMetadata": null
+        "json": {
+            "timestampMillis": 1640692800000,
+            "partitionSpec": {
+                "type": "FULL_TABLE",
+                "partition": "FULL_TABLE_SNAPSHOT"
+            },
+            "runId": "2021-12-28T12:00:00Z",
+            "assertionUrn": "urn:li:assertion:6a54a953afa6403d7b6dffbf02ff3de4",
+            "asserteeUrn": "urn:li:dataset:(urn:li:dataPlatform:postgres,postgres1.postgres.public.foo2,PROD)",
+            "batchSpec": {
+                "customProperties": {
+                    "data_asset_name": "foo2",
+                    "datasource_name": "my_postgresql_datasource"
+                },
+                "nativeBatchId": "010ef8c1cd417910b971f4468f024ec5"
+            },
+            "status": "COMPLETE",
+            "result": {
+                "type": "SUCCESS",
+                "rowCount": 5,
+                "unexpectedCount": 0,
+                "nativeResults": {}
+            }
+        }
+    }
+},
+{
+    "entityType": "assertion",
+    "entityUrn": "urn:li:assertion:fe1fad936d3dcb98a501525c66682253",
+    "changeType": "UPSERT",
+    "aspectName": "assertionInfo",
+    "aspect": {
+        "json": {
+            "customProperties": {
+                "expectation_suite_name": "test_suite"
+            },
+            "type": "DATASET",
+            "datasetAssertion": {
+                "dataset": "urn:li:dataset:(urn:li:dataPlatform:postgres,postgres1.postgres.public.foo3,PROD)",
```

Comment:
These are new mcps - assertions on table with name FOO3 and urn is urn:li:dataset:(urn:li:dataPlatform:postgres,postgres1.postgres.public.foo3,PROD)